release/action.yaml

```diff
@@ -44,7 +44,7 @@ runs:
   steps:
     - name: Checkout
       uses: actions/checkout@v3
-      if: inputs.checkout-repo
+      if: ${{ inputs.checkout-repo == 'true' }}
```

Comment:
Just a minor thing. You don't need the `${{ }}` and can just put `inputs.checkout-repo == 'true'`

Reply:
See https://github.com/open-turo/action-renovate/blob/3e4bfc349d2a5bfc4314053ed46a7854968802c5/action.yaml#L67

Reply:
Updated ✅